content/css/concepts/filter-functions/terms/contrast/contrast.md

```diff
@@ -38,3 +38,6 @@ Give the image a contrast of `200%`:
   filter: contrast(200%);
 }
 ```
+![Contrast example](/media/css-filterfunctions-contrast-example.png)
+
+Here, the original image is on the left and the image on the right has the filter set at `contrast(200%)`
```

Comment:
Since you've added this line here, you can remove the text that reads "Before" and "After" above the images in `css-filterfunction-contrast-example.png`.

Reply:
Changes done and CLA signed.